x-pack/plugins/maps/public/plugin.ts

```diff
@@ -179,10 +181,13 @@ export class MapsPlugin
       euiIconType: APP_ICON_SOLUTION,
       category: DEFAULT_APP_CATEGORIES.kibana,
       async mount(params: AppMountParameters) {
+        const [coreStart, pluginsStart] = await core.getStartServices();
+        const { http } = coreStart;
+        const { spaces: spacesApi } = pluginsStart as MapsPluginStartDependencies;
         const UsageTracker =
           plugins.usageCollection?.components.ApplicationUsageTrackingProvider ?? React.Fragment;
         const { renderApp } = await lazyLoadMapModules();
-        return renderApp(params, UsageTracker);
+        return renderApp(params, UsageTracker, http, spacesApi);
```

Comment:
Not sure passing spaces into renderApp this way is needed. Spaces is a `start` dependency and can just be grabbed from kibana_services when needed to align with how all `start` dependencies are obtained.